lib/mongorito.js

```diff
@@ -25,6 +16,14 @@ const is = require('is_js');
 
 const Query = require('./query');
 
+const MongoClient = mongodb.MongoClient;
+
+class EnforcerError extends Error {
```

Comment:
This and the instance Symbol enforce a single mongoritoInstance, the assumed this instance was failing all over the place for me in Node.js 4.1+ so I switched to a real Singleton pattern which is what this is enforcing.